oml/models/vit/clip.py

```diff
@@ -0,0 +1,240 @@
+from collections import OrderedDict
```

Comment:
please, add the comment with the link to the original source code
and let's keep the 3d party code and our code separately as we did it for ViT (there are 3 files under the vit directory, but only 1 of them contains our code)

Reply:
Ok, sure